.travis.yml

```diff
@@ -7,17 +7,17 @@ env:
   - DO_LINT=true DO_UNIT=true DO_FUNCTIONAL=false
   - DO_LINT=false DO_UNIT=false DO_FUNCTIONAL=true
```

Comment:
Yeah that's something I was going back and forth on. I kind of just want to run both forever -- then we always get that extra sanity check that ClusterRunner fails when it should and passes when it should.